splunk/authorization_flow_dashboard.xml

```diff
@@ -357,7 +357,7 @@
       <chart>
         <title>12. Authorized Application Receives Token Events TIME-CHART</title>
         <search base="baseSearch12">
-          <query>timechart count</query>
+          <query>search message.auth_grant_type=authorization_code | stats count</query>
```

Comment:
This looks really good - one question, I could not get any results on 12. Authorized Application Receives Token Events TIME-CHART - was wondering if that should remain a `| timechart count`? I opened the query in a search and a `| stats count` returns 4000+

Reply:
@oragame Good catch!  I broke it :-) Just pushed up a fix changing it back to `| timechart count`.